src/main/java/sg/ncl/domain/MachineType.java

```diff
@@ -0,0 +1,8 @@
+package sg.ncl.domain;
+
+/**
+ * @author Te Ye
+ */
+public enum MachineType {
+    SystemX
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00115)